region/maridia/outer/Main Street.json

```diff
@@ -5117,7 +5118,7 @@
         "canInsaneJump",
         "canReserveDoubleDamageBoost",
         {"or": [
-          "h_maxHeightSpringBallJump",
+          "h_underwaterMaxHeightSpringBallJump",
```

Comment:
The springballjump part isn't any harder.  I had to relearn how to do it  is all.
The blind damageboost with a spikesuit sounds scary; I didn't test that.

Reply:
I wasn't really sure what to do with this - i think it would look how I wrote it, but since it hasn't been tested, I just put flash suit checked = false